static/js/subscription.js

```diff
@@ -0,0 +1,27 @@
+/*eslint-env jquery*/
+/*eslint semi: ["error", "always"]*/
+
+
+$(document).ready(function() {
+  $('.posted-form').submit(function (e) {
+    e.preventDefault();
+    var form = e.target;
+    $.ajax({
+      headers: {
+        'Content-Type': 'application/x-www-form-urlencoded'
+      },
+      type: form.method,
+      url:  form.action,
+      dataType: 'json',
+      processData: false,
+      data: $(this).serialize(),
+      success: function (data) {
+        form.reset();
+        //TODO NEED TO USE THAT BLOCK FOR MESSAGES.
```

Comment:
Thinking to use [bootstrap-Toast](https://getbootstrap.com/docs/4.2/components/toasts/) for that